CONTRIBUTING.md

```diff
@@ -0,0 +1,124 @@
+# Contribution Guide
+
+Thank you for your interest in contributing to the Instill AI Cookbook! This guide will help you get started with contributing to the project by explaining the process for submitting your code, guidelines for code quality, and best practices for writing Jupyter notebooks.
+
+## Table of Contents
+1. [How to Contribute](#how-to-contribute)
+2. [Code Quality Standards](#code-quality-standards)
+3. [Best Practices for Jupyter Notebooks](#best-practices-for-jupyter-notebooks)
+4. [Updating the Repository README](#updating-the-repository-readme)
+
+---
+
+## How to Contribute
+
+### 1. Reporting Issues
+If you encounter a bug, have a suggestion, or want to request a new notebook, please open an issue in the repository. When submitting an issue:
+- Provide a clear, descriptive title.
+- Include as much detail as possible (steps to reproduce, screenshots, etc.).
+- Suggest possible solutions if applicable.
```

Comment:
```suggestion
If you encounter a bug, have a suggestion, or want to request a new notebook, please open an issue in the [instill-core](https://github.com/instill-ai/instill-core/issues/new/choose) repository. When submitting an issue:
- Select an appropriate template.
- Provide a clear, descriptive title.
- Include as much detail as possible (steps to reproduce, screenshots, etc.).
- Suggest possible solutions if applicable.
```

@GeorgeWilliamStrong By opening issue I think we mean in the instill-core project right? If so, please check my suggestion above 🙏